.github/workflows/reusable-dir-demo.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright AGNTCY Contributors (https://github.com/agntcy)
+# SPDX-License-Identifier: Apache-2.0
+
+name: Demo
+
+description: |
+  This workflow demonstrates how to use the Dir CLI with a Kind cluster and Helm chart.
+  It includes steps for building, pushing, publishing, and listing images.
+
+on:
+  workflow_dispatch:
```

Comment:
I think, if it is as a reusable workflow, it should be called from an other workflow. So it should be `workflow_call` instead of `workflow_dispatch`

Reply:
I will change the `reusable-*` naming then because I think this is something we want to run ad-hoc to demo the agent flow